alf/utils/summary_utils.py

```diff
@@ -165,6 +166,59 @@ def add_nested_summaries(prefix, data):
             alf.summary.scalar(name, elem)
 
 
+@_summary_wrapper
+@alf.configurable
+def summarize_per_category_loss(loss_info: LossInfo,
+                                summarize_count=False,
+                                label_name: Optional[List[str]] = None):
+    """Add summary about each category of the unaggregated ``loss_info.loss``
+    of the shape (T, B), or (B, ) by partationing it according to
+    ``loss_info.batch_label``, which has the same shape as ``loss_info.loss``.
+    It also creates summarization of the number of samples encountered
+    for each category.
+
+    Args:
+        loss_info (LossInfo): do per-category summarization if
```

Comment:
No need to annotate type if type hints are provided in the function signature.

Reply:
Removed unnecessary type annotations.